examples/internal/variables.tf

```diff
@@ -0,0 +1,44 @@
+variable "region" {
+  description = "The name of an AWS region"
+  type        = string
+}
+
+variable "env" {
+  description = "Resource environment tag (i.e. dev, stage, prod)"
+  type        = string
+  default     = "test"
+}
+
+variable "service" {
+  description = "Resource service tag"
+  type        = string
+  default     = "egress-proxy"
+}
+
+variable "description" {
```

Comment:
Could this variable be named a bit better? `description_tag` or similar

Reply:
it wasn't in use, so I removed it